src/main/resources/application.properties

```diff
@@ -1 +1,14 @@
+spring.profiles.active=dev
 
+#!--- this is an example to your self enviroment application-{EXAMPLE}.properties file
+# this configuration is for MySql database with JDBC/JPA
+#spring.datasource.url=jdbc:mysql://localhost:3306/testhotels
+#spring.datasource.username=xxxx
+#spring.datasource.password=xxxx
+#spring.datasource.driver.class=com.mysql.jdbc.Driver
+#spring.jpa.hibernate.ddl-auto=update
+#spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5InnoDBDialect
+#!---Logging properties (Uncomment and use next two lines to debug JPA methods)
+#logging.level.org.hibernate.SQL=DEBUG
+#logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
+#---!
```

Comment:
https://stackoverflow.com/questions/35531661/using-env-variable-in-spring-boots-application-properties

Reply:
Lei los comentarios y en general indican que deben generarse los app properties para cada entorno, por algo en particular crees que es mejor usar variables de entorno que disitinos app properties?